invoices/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+
+class Invoice(models.Model):
+    """ Model for invoices
+    """
+    DUE_RECEIPT = 'Due of Receipt'
```

Comment:
@cholorecto check my comment on this. I repeated this part 2 times already. this will the the 3rd